asn1tools/rvcs.py

```diff
@@ -106,28 +107,50 @@ def cmd_test(args):
 
         print("%s is %dB in UPER" % (path, len(uper)))
 
-def main():
-    default_schema = os.path.join(
-        os.path.dirname(os.path.realpath(__file__)),
-        "../schema.asn")
+schema_help_String  = 'Specify the addtional ASN.1 schema files to the standard\n' +\
+                      'ones under /schema directory. Multiple ASN.1 schema files\n' +\
+                      'could be specified using \',\' as the separator.\n' +\
+                      'Example: --schema ../myschema1.asn,../myschema2.asn or\n'+\
+                      '         --schema absolute_dir/myschema3.asn\n'
+convert_help_string = 'Convert between binary and human-readable formats.'
+test_help_string    = 'Test encoding/decoding the given files.'
 
-    parser = argparse.ArgumentParser()
-    parser.add_argument('--schema', default=default_schema)
+def main():
+    # Walk through ../schema for all standard ASN.1 schema files.
+    default_schema = ''
+    for root, dirs, schemafiles in os.walk(os.path.join (
+        os.path.dirname(os.path.realpath(__file__)), "../schema")):
+        for schemafile in schemafiles:
+            if default_schema != "":
+                default_schema = default_schema + ','
+            default_schema = default_schema + root + '/' + schemafile
+
+    parser = argparse.ArgumentParser(formatter_class=RawTextHelpFormatter)
+    parser.add_argument('--schema', default=default_schema,
```

Comment:
This really should be a list. I think you can do it by adding nargs='*'. Then we don't need to worry about dealing with strings with commas in them everywhere.

Reply:
I found that if we use "nargs=" and have the white space between the input schema files, the following arguments (such as "convert", source jer file name, and target binary file name) are also count as the arguments to "--schema". Unless we put "-schema" as the last option to rvcs.py. Any suggestions?